### PR TITLE
Add predictions to dashboard

### PR DESCRIPTION
**Pull request by zuzg**
image
It is ugly but it works. TBD:
* SMILES graph (there is matplotlib tool but it crashes with dash)
* styling

### CODE REVIEW

**Comment by zuzg** [comment]
Update: 
image

I'm terrible at html, sorry 😭

File: dashboard/pages/hit_validation/callbacks.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import base64
+import dash_dangerously_set_inner_html as dhtml
```
Comment (Tremirre) [inline dashboard/pages/hit_validation/callbacks.py:2]: I believe this is missing from requirements.txt

File: dashboard/visualization/plots.py [hunk 2]
```diff
@@ -1,10 +1,16 @@
 from itertools import product
 
+import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
 import plotly.express as px
 import plotly.graph_objects as go
+import plotly.tools as tls
 from plotly.subplots import make_subplots
+from rdkit import Chem
```
Comment (Tremirre) [inline dashboard/visualization/plots.py:10]: Same here

**Review by Tremirre — APPROVED** [review]
Seems OK, you just need to add the dependencies (btw imo we could store the predictions as CSV file - easier for browsing and more to liking of git but that's a minor thing)

File: dashboard/visualization/plots.py (hunk 2 above)
Comment (zuzg) [reply dashboard/visualization/plots.py:10]: We already have rdkit

**Pull request merged**